addons/mail/static/src/models/thread/thread.js

```diff
@@ -2293,6 +2319,18 @@ function factory(dependencies) {
             compute: '_computeIsCurrentPartnerFollowing',
             default: false,
         }),
+        /**
+         * States whether this thread description is editable by the current user.
+         */
+        isDescriptionEditableByCurrentUser: attr({
+            compute: '_computeIsDescriptionEditableByCurrentUser',
+        }),
+        /**
+         * States whether this thread description is highlighted.
+         */
+        isDescriptionHighlighted: attr({
+            compute: '_computeIsDescriptionHighlighted'
+        }),
```

Comment:
This field should be on thread view topbar as it depends on `isMouseOverThreadDescription`.